src/views/components/Lists/ListsLists/__tests__/container.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import configureStore from 'redux-mock-store'
+
+import diveTo from 'TestUtils/diveToEnzyme'
+
+import ListsLists, { ListsListsContainer } from '../container'
+
+jest.mock('react-intl', () => ({
+  get injectIntl() {
```

Comment:
why get?

Reply:
We can call with you and you try to help me:) InjectIntl very strange